templates/keepalived.conf.j2

```diff
@@ -167,6 +170,15 @@ virtual_server {{ vserver.ip }} {{ vserver.port }} {
 
   {% for rserver in vserver.real_servers %}
   real_server {{ rserver.ip }} {{ rserver.port }} {
+    {% if rserver.weight is defined and rserver.weight | int %}
+    weight {{ rserver.weight }}
```

Comment:
weight being an integer, you can force it here in L174 by adding | int.

Reply:
But it's already "forced" to be an INT in L173
Do you want these options to fail if it's not INT like the `advert_int` ?

Reply:
It's not "forced" to be an int on L173.

In fact, we should probably remove the ``and rserver.weight | int`` part of the condition, as it is allowed to have 0 as weight.

However, I think we should have `weight {{ rserver.weight | int }}`  at L174, to force the templating of an integer.